bookkeeper-server/src/main/proto/BookkeeperProtocol.proto

```diff
@@ -136,6 +147,11 @@ message GetBookieInfoRequest {
     optional int64 requested = 1;
 }
 
+message SyncRequest {
+   required int64 ledgerId = 1;
```

Comment:
Why is the ledgerId included at all? Surely when you send it you expect all the journal up until this message to be fsynced.

Reply:
because this API means 'force and wait for a sync on data for the given ledger"
A bookie has several journals and we want to fsync data only on the journal assigned to the ledger (just for an example).

If you do not have "journal" with this primitive the clients want the bookie to acknowledge that data sent to him for the ledger has been persisted durably

Reply:
@ivankelly it is a per ledger behavior. the bookie storage implementation can either use one journal, multiple journals or bypassing journal.